code/game/gamemodes/blob/overmind.dm

```diff
@@ -43,8 +42,7 @@
 	..()
 
 /mob/camera/blob/Destroy()
-	overminds -= src
```

Comment:
that's a mistake, I didn't fix the merge conflicts correctly, will fix that in the real PR.